srcpkgs/cquery/template

```diff
@@ -0,0 +1,29 @@
+pkgname="cquery"
```

Comment:
add the template header on top of template like so

# Template file for 'cquery'

Reply:
Fixed.